Dockerfile

```diff
@@ -1,34 +1,34 @@
-FROM centos:8
+FROM centos/python-38-centos7
 
 LABEL description="coldfront"
 
-# install dependencies
-RUN yum -y install epel-release
-RUN yum -y update
-RUN yum -y install python36 python36-devel git memcached redis
-
+USER root
 WORKDIR /root
-
-# install coldfront
-RUN mkdir /opt/coldfront_app
-
-WORKDIR /opt/coldfront_app
-
-RUN cd /opt/coldfront_app
-RUN git clone https://github.com/ubccr/coldfront.git
-RUN python3.6 -mvenv venv
-RUN source venv/bin/activate
-
-WORKDIR /opt/coldfront_app/coldfront
-
-RUN cd /opt/coldfront_app/coldfront
-RUN pip3 install wheel
-RUN pip3 install -r requirements.txt
-RUN cp coldfront/config/local_settings.py.sample coldfront/config/local_settings.py
-RUN cp coldfront/config/local_strings.py.sample coldfront/config/local_strings.py
-
-RUN python3 ./manage.py initial_setup
-RUN python3 ./manage.py load_test_data
-
-EXPOSE 8000
+COPY requirements.txt ./
+RUN pip install -r requirements.txt && rm requirements.txt
+RUN pip install jinja2 pyyaml
+
+# mybrc or mylrc
+ARG portal="mybrc"
+RUN mkdir -p /var/log/user_portals/cf_${portal} \
+ && touch /var/log/user_portals/cf_${portal}/cf_${portal}_{portal,api}.log \
+ && chmod 775 /var/log/user_portals/cf_${portal} \
+ && chmod 664 /var/log/user_portals/cf_${portal}/cf_${portal}_{portal,api}.log
+
+ARG base_dir="/vagrant/coldfront_app"
```

Comment:
This path is hardcoded in a volume in `docker-compose.yml` but configurable here. I'd say we can keep it hardcoded.